apps/wakunode2/spammer.nim

```diff
@@ -0,0 +1,119 @@
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
+
+import
+  chronos,
+  chronicles,
+  stew/[byteutils, results],
+  std/times,
+  libp2p/protocols/pubsub/gossipsub,
+  strutils
+
+import
+  ../../waku/factory/waku,
+  ../../waku/factory/external_config,
+  ../../waku/waku_core,
+  ../../waku/waku_relay,
+  ../../waku/node/waku_node,
+  ../../waku/node/peer_manager/peer_manager,
+  ../../waku/waku_rln_relay/rln_relay,
+  ../../tests/waku_rln_relay/rln/waku_rln_relay_utils
+
+proc send(
+    waku: Waku, contentTopic: ContentTopic
+): Future[Result[void, string]] {.async.} =
+  var ephemeral = true
+
+  var message = WakuMessage(
+    payload: toBytes("Hello World!" & intToStr(int(getTime().toUnix()))),
+    contentTopic: contentTopic,
+    #      meta: metaBytes,
+    version: 2,
+    timestamp: getNanosecondTime(getTime().toUnixFloat()),
+    ephemeral: ephemeral,
+  )
+
+  let rlnRes =
+    waku.node.wakuRlnRelay.unsafeAppendRLNProof(message, float64(getTime().toUnix()))
+  if rlnRes.isOk:
+    let pubRes = await waku.node.publish(none(PubsubTopic), message)
+    if pubRes.isErr():
+      error "failed to publish", msg = pubRes.error
+      return err(pubRes.error)
+  else:
+    error "failed to append RLNProof", err = rlnRes.error
+    return err(rlnRes.error)
+
+  return ok()
+
+proc burstPublish(
+    waku: Waku, conf: WakuNodeConf, contentTopic: ContentTopic
+) {.async.} =
+  var futures: seq[Future[Result[void, string]]]
+  var i: uint64 = 0
+  var start = getTime().toUnixFloat()
+
+  while i < conf.rlnRelayUserMessageLimit:
```

Comment:
maybe `i` not needed?
and here
for i in conf.rlnRelayUserMessageLimit

Reply:
This is not how Nim works:-P There is a function I could use to turn the msg limit into iterator, but I am not sure it is worth searching for it:D

Reply:
ah yeah my bad. gues for i in 0..<conf.rlnRelayUserMessageLimit 
no big deal tho